README.md

```diff
@@ -12,8 +12,9 @@ Alternatively, see _Running tests in ephemeral namespaces_ below.
 All tests will run in the project you're logged into, so it should be empty.
 If there are resources deployed in the project, you should not expect they will survive.
 
-Running the tests amounts to standard `mvn clean verify`.
-If the test suite depends on Quarkus `999-SNAPSHOT` (it does currently), make sure you have built Quarkus locally prior to running the tests.
+Running the tests amounts to standard `mvn clean verify` or `mvn clean verify -Dquarkus-master`.
+Certain Quarkus version can be specified using `version.quarkus` property.
+If the test suite run depends on Quarkus `999-SNAPSHOT`, make sure you have built Quarkus locally prior to running the tests.
```

Comment:
```
Running the tests amounts to standard `mvn clean verify`.
This will use a specific Quarkus version, which can be modified by setting the `version.quarkus` property.
Alternatively, you can use `-Dquarkus-master` to run the test suite against Quarkus `999-SNAPSHOT`.
In that case, make sure you have built Quarkus locally prior to running the tests.
```

Reply:
Suggestion applied